.github/workflows/dailies.yml

```
@@ -0,0 +1,27 @@
+name: Daily workflows
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 16 * * *"  # Daily at noon EST
+
+jobs:
+
+  run-daily-tests:
+    uses: catalystneuro/tqdm_publisher/.github/workflows/testing.yml@main
+
+  notify:
+    runs-on: ubuntu-latest
+    needs: [run-daily-tests]
+    if: ${{ always() && contains(needs.*.result, 'failure') }}
+    steps:
+    - uses: dawidd6/action-send-mail@v3
+      with:
+        server_address: smtp.gmail.com
+        server_port: 465  # TSL
+        username: ${{ secrets.MAIL_USERNAME }}
+        password: ${{ secrets.MAIL_PASSWORD }}
+        subject: NWB Inspector Daily Failure
+        to: [garrett.flynn@catalystneuro.com, cody.baker@catalystneuro.com]  # add more with commas, no separation
```

From https://github.com/dawidd6/action-send-mail?tab=readme-ov-file#usage, no brackets surrounding I think

```suggestion
        to: garrett.flynn@catalystneuro.com,cody.baker@catalystneuro.com  # add more with commas, no separation
```